django/forms/fields.py

```diff
@@ -1189,7 +1189,7 @@ class UUIDField(CharField):
 
     def prepare_value(self, value):
         if isinstance(value, uuid.UUID):
-            return value.hex
```

Comment:
This lines dates back to the original UUID patch in ed7821231b7dbf34a6c8ca65be3b9bcbda4a0703.

I didn't spot anything in https://code.djangoproject.com/ticket/19463 discussing this.

This change change looks reasonably safe to me but may be worth a line in the release notes.

Reply:
Now, `show_with_dashes` argument is added, make it more safety.